0000-catalog.md

```diff
@@ -0,0 +1,427 @@
+PropertyValues are in addition to the expected PropertyValues of a [Grid GS1 
+Product](https://github.com/hyperledger/grid-rfcs/blob/fbedec06d70b16492fea9f6b1e87146c5fc56771/0000-product.md).
+
+GTIN formats consist of 14-digit “numeric strings” which include some amount of
+internal “0” padding depending on the specific GTIN format (GTIN-12,
+GTIN-13, or GTIN-14).  After the 12-hex-characters that are consumed by the grid
+namespace prefix, the product namespace prefix, GS1 namespce prefix, and catalog 
+product namespace prefix, there are 58 hex characters remaining in the address.  
+The 12 to 14 digits of the GTIN can be left padded with 40 to 42-hex-character zeroes and 
+right padded with 2-hex-character zeroes to accommodate potential future storage 
+associated with the GS1 Product representation, for example:
+
+``` 
+“621dee” + “02” + “01” + "7d1734adfee4927" + “00000000000000000000000000000” 
++ 14-character “numeric string” product_id + “00” // product_id == GTIN 
+```
+
+A full GS1 Product address using the example GTIN from https://www.gtin.info/
+would therefore be:
+
+``` 
+“621dee02017d1734adfee4927000000000000000000000000000000001234560001200” 
+```
+
+### Catalog Product Schema Definition
+
+This schema defines the additional attributes required of a grid product, such that 
+it may be considered a "catalog product". Catalog products should be grouped with 
+other catalog products. Once grouped, the grid catalog can be shared with other 
+participants in a grid network.
+
+
+Any grid product added to a catalog will need to contain a _required_ ***status*** 
+field and can include an _optional_ ***prices*** field, which will be enforced by 
+the catalog_product grid schema.
+
+```
+Schema(
+    name="Catalog Product",
+    description="A grid product that can be grouped with other products via grid catalog",
+    owner = "Target"
+    properties=[
+        PropertyDefinition(
+            name="catalog_id",
+            data_type=PropertyDefinition.DataType.STRING,
+            description="The the id of the catalog this 'catalog product' belongs to",
+            required=True
+        ),
+        PropertyDefinition(
+            name="status",
+            data_type=PropertyDefinition.DataType.ENUM,
+            description="The current state of the catalog product",
+            enum_options=["ACTIVE", "INACTIVE"],
+            required=True
+        ),
+        PropertyDefinition(
+            name="price",
+            data_type=PropertyDefinition.DataType.STRING,
+            description="The price of the catalog product"
+            required=False
+        )])
+```
+
+In order to store a catalog product in global state, we will extend from the 
+definition of a grid product with several fixed fields, namely identifiers, as 
+well as the set of dynamic property values that will conform to the catalog 
+product schema definition.
+
+```
+message Product { 
```

Comment:
This doesn't match the example below

Reply:
✅

Reply:
I think I understand what I was missing here: you referenced Product from the product RFC, and the give an example of `CatalogProduct`, but haven't defined the proto for `CatalogProduct`.

Took me a bit to figure that out because one of the fields in the example has some whitespace/formatting issues that made it look like it was part of a trailing comment.